schema/v0_1_0/mzqc_0_1_0.schema.json

```diff
@@ -0,0 +1,240 @@
+					"description": "List of runQuality elements.",
+					"type": "array",
+					"minItems": 1,
+					"items": {
+						"$ref": "#/definitions/runQuality"
+					}
+				},
+				"setQualities": {
+					"description": "List of setQuality elements.",
+					"type": "array",
+					"minItems": 1,
+					"items": {
+						"$ref": "#/definitions/setQuality"
+					}
+				},
+				"controlledVocabularies": {
+					"description": "Collection of CV elements.",
+					"type": "array",
+					"minItems": 1,
+					"items": {
+						"$ref": "#/definitions/controlledVocabulary"
+					}
+				}
+			},
+			"additionalProperties": false,
+			"anyOf": [
+				{"required": ["runQualities"]},
+				{"required": ["setQualities"]}
+			],
+			"required": ["version", "controlledVocabularies"]
+		}
+	},
+	"additionalProperties": false,
+	"required": ["mzQC"],
+	"definitions": {
+		"baseQuality": {
+			"description": "Base element describing a runQuality or setQuality.",
+			"type": "object",
+			"properties": {
+				"metadata": {
+					"$ref": "#/definitions/metadata"
+				},
+				"qualityMetrics": {
+					"type": "array",
+					"minItems": 1,
+					"items": {
+						"$ref": "#/definitions/qualityMetric"
+					}
+				}
+			},
+			"additionalProperties": false,
+			"required": ["metadata", "qualityMetrics"]
+		},
+		"runQuality": {
+			"description": "Parent element describing QC metrics for a single run.",
+			"$ref": "#/definitions/baseQuality"
+		},
+		"setQuality": {
+			"description": "Parent element describing combined QC metrics for multiple runs.",
+			"$ref": "#/definitions/baseQuality"
+		},
+		"cvParameter": {
+			"description": "Base element containing a reference to a controlled vocabulary.",
+			"type": "object",
+			"properties": {
+				"cvRef": {
+					"description": "Reference to the CV that contains the parameter definition.",
+					"type": "string",
+					"pattern": "^[A-Z]+$"
+				},
+				"accession": {
+					"description": "Accession number identifying the parameter within its CV.",
+					"type": "string",
+					"pattern": "^[A-Z]+:[0-9]{7}$"
+				},
+				"name": {
+					"description": "Name of the CV element describing the parameter.",
+					"type": "string"
+				},
+				"description": {
+					"description": "Description of the parameter.",
+					"type": "string"
+				},
+				"value": {
+					"description": "Value of the parameter."
+				},
+				"unit": {
+					"description": "A CV element describing the unit of the parameter value.",
+					"anyOf": [
+						{
+							"$ref": "#/definitions/cvParameter"
+						},
+						{
+							"type": "array",
+							"minItems": 1,
+							"items": {
+								"$ref": "#/definitions/cvParameter"
+							}
+						}
+					]
+				}
+			},
+			"required": ["cvRef", "accession", "name"]
+		},
+		"metadata": {
+			"description": "Metadata describing the QC analysis.",
+			"type": "object",
+			"properties": {
+				"fileProvenance": {
+					"description": "Optional string describing/documenting the provenance of the file",
```

Comment:
This is pretty vague. Whereas proper file provenance is very important, the way this property is right know I don't know how useful it'd be.

Reply:
I agree that it is important for a number of use cases. So, it was optional. Would you rather abolish it completely or reshape/refine it.

Reply:
We had a discussion during the call on that. It seems this provenance can be fairly complex, so we better take the decision to not include provenance even as optional in the schema, rather leave it for a later revision of the format with a focus on this particular feature. Like that, we will not have conflicting 'objects of provenance'.